pennylane/templates/embeddings.py

```diff
@@ -38,6 +38,17 @@ def AmplitudeEmbedding(features, wires, pad=None, normalize=False):
     The L2-norm of ``features`` must be one. By default, ``AmplitudeEmbedding`` expects a normalized
     feature vector. The argument ``normalize`` can be set to ``True`` to automatically normalize it.
 
+    .. note::
+
+        ``AmplitudeEmbedding`` uses PennyLane's :class:`~pennylane.ops.QubitStateVector`
+        and only works in conjunction with devices that implement this operation.
+
```

Comment:
Also added this note...

Reply:
This is not so true, as ``AmplitudeEmbedding`` should automatically fallback to `Mottonen` if `QubitStateVector` is not implemented, so this should work on almost any device. I think this note can be removed.